airbyte-integrations/connectors/source-amazon-ads/acceptance-test-config.yml

```diff
@@ -0,0 +1,77 @@
+# See [Source Acceptance Tests](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector/source-acceptance-tests.md)
+# for more information about how to configure these tests
+connector_image: airbyte/source-amazon-ads:dev
+tests:
+  spec:
+    - spec_path: "integration_tests/spec.json"
+  connection:
+    - config_path: "secrets/config.json"
+      status: "succeed"
+    - config_path: "integration_tests/invalid_config.json"
+      status: "failed"
+  discovery:
+    - config_path: "secrets/config.json"
+  basic_read:
+    - config_path: "secrets/config.json"
+      configured_catalog_path: "integration_tests/configured_catalog_products.json"
+      empty_streams:
+        [
+          "sponsored_product_negative_keywords",
+          "sponsored_product_targetings",
+          "sponsored_product_ads",
+        ]
+      expect_records:
+        path: "integration_tests/expected_produts_stream.txt"
+        extra_fields: no
+        exact_order: no
+        extra_records: yes
+
+    - config_path: "secrets/config.json"
+      configured_catalog_path: "integration_tests/no_profiles_catalog.json"
+      empty_streams:
+        ["sponsored_display_product_ads", "sponsored_display_targetings"]
+    - config_path: "secrets/config.json"
+      configured_catalog_path: "integration_tests/configured_catalog_display.json"
+      empty_streams:
+        [
+          "display_report_stream",
+          "sponsored_display_product_ads",
+          "sponsored_display_targetings",
+        ]
+      expect_records:
+        path: "integration_tests/expected_display_stream.txt"
+        extra_fields: no
+        exact_order: no
+        extra_records: yes
+    - config_path: "secrets/config.json"
+      configured_catalog_path: "integration_tests/configured_catalog_brands.json"
+      empty_streams: ["sponsored_brands_report_stream"]
+      expect_records:
+        path: "integration_tests/expected_brands_stream.txt"
+        extra_fields: no
+        exact_order: no
+        extra_records: no
+    - config_path: "secrets/config.json"
+      configured_catalog_path: "integration_tests/configured_catalog.json"
```

Comment:
If we run the `basic_read `test separately for each directory, then why run one common one at the end?

Reply:
Oleh suggested to do this and I thought it is a good idea. Do you think I should undo this or delete separate test for brands, products and displays streams?

Reply:
I suggested just to add file `integration_tests/configured_catalog.json`, not to use this file in `basic_read`.

Reply:
ok, will remove it from basic read tests.